pkg/agent/openflow/multicast.go

```diff
@@ -77,6 +79,18 @@ func (f *featureMulticast) replayFlows() []binding.Flow {
 	return getCachedFlows(f.cachedFlows)
 }
 
+// multicastIngressPodMetricFlows generates the flows to forward multicast ingress packets before
+// outputting to MulticastOutputTable. It matches TargetOFPortField with port number to specify
```

Comment:
"with the OFPort of a multicast receiver Pod"?

Reply:
updated